apps/web/lib/clients/blockchain/docs/usage-patterns.md

```
@@ -0,0 +1,447 @@
+# Blockchain Services Usage Patterns
+
+This document provides comprehensive examples and best practices for integrating with the Mock Blockchain Services.
+
+## Table of Contents
+
+1. [Service Initialization](#service-initialization)
+2. [Dependency Injection Patterns](#dependency-injection-patterns)
+3. [Event Handling and Subscriptions](#event-handling-and-subscriptions)
+4. [Testing Strategies](#testing-strategies)
+5. [Error Handling Best Practices](#error-handling-best-practices)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)